lib/eco/inclexer/inclexer.py

```diff
@@ -515,8 +515,10 @@ def relex(self, node):
             try:
                 token = next_token()
                 lookaheads.append(token[2])
-                if token[3][0] is startnode:
-                    past_startnode = True
+                for temp in token[3]:
```

Comment:
So better left as-is or ... ? [I don't mind, but just want to be sure you've thought about it!]